spec/features/lease_spec.rb

```diff
@@ -0,0 +1,46 @@
+RSpec.feature 'leases' do
+  let(:user) { create(:user) }
+  before do
+    sign_in user
+  end
+  describe 'create a new leased object' do
+    let(:future_date) { 5.days.from_now }
+    let(:later_future_date) { 10.days.from_now }
+
+    # Pending revision to work in hyrax
+    # TODO: https://github.com/samvera-labs/hyrax/issues/1123
```

Comment:
I don't think this is acceptable. We need to make the tests pass in order to merge this work in. I suspect the are not passing because the tests are written with the assumption that it's starting from an empty repository.  If you add `:clean_repo` to the spec metadata, that may fix it.